### PR TITLE
DOCS-3291 Adds Mobile and Performance Dashboards Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds two doc pages and screenshots for the mobile and performance dashboards, updates the menu navigation, and edits the overall Dashboards section.

### Motivation

DOCS-3291

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by urseberry** [comment]
https://datadoghq.atlassian.net/browse/DOCS-3692 created for editorial review

File: content/en/real_user_monitoring/dashboards/_index.md [hunk 1]
```diff
@@ -43,9 +46,9 @@ To explore individual events, click on a graph and click **View RUM events**. Th
 
 ### Customize dashboards
 
-Clone your RUM dashboards and customize them to fit your needs. You can add widgets and modify the template variables.
+To clone your RUM dashboards, click the click the **Settings** icon and select **Clone dashboard**. To add more widgets, powerpacks, or apps, scroll down to the bottom and click the **+** icon.
```
Comment (maycmlee) [inline content/en/real_user_monitoring/dashboards/_index.md:49]:
```suggestion
To clone your RUM dashboards, click the **Settings** icon and select **Clone dashboard**. To add more widgets, powerpacks, or apps, scroll down to the bottom and click the **+** icon. 
```

File: content/en/real_user_monitoring/dashboards/user_sessions_dashboard.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+It shows:
+
+- **Sessions**: Analyze the number of user sessions, the average session duration, and the number of page views per session. Two graphs on the right visualize the number of sessions and the top pages. 
+- **User demographics**: Observe the number of sessions by country and the top countries, devices, and operating systems for your application. You can also view a graph of the top browser usage shares. 
+- **Application usage**: See graphs of the average session duration, page views per sessions, actions per session, and errors per session. The tables below list usage metrics based on the first and last visited pages.
+
+{{< img src="real_user_monitoring/dashboards/sessions_dashboard.png" alt="Out-of-the-box User sessions dashboard" style="width:100%;" >}}
+
+For more information about the information displayed, see [RUM Browser Data Collected][2].
```
Comment (maycmlee) [inline content/en/real_user_monitoring/dashboards/user_sessions_dashboard.md:22]:
```suggestion
For more information about the data displayed, see [RUM Browser Data Collected][2].
```

**Review by maycmlee — APPROVED** [review]
Just a couple of small edits!

**Pull request merged**